tests/integration/test_integration.py

```diff
@@ -22,29 +22,29 @@
 
 @pytest.fixture(scope="module")
 def client():
-    return cellengine.APIClient("gegnew", "testpass123")
+    return cellengine.APIClient("gegnew", "^w^A7kpB$2sezF")
```

Comment:
Is this for cellengine.com or your local server? If you need a cellengine.com password for integration tests, it needs to go in the GitHub secrets list (and this password needs to be reset).

Reply:
Whoooop, just had this for local development, forgot to remove it. Password changed. I _will_ add a GH Action for running the integration tests, at which points I will need a test user and password for CE in GH Secrets. Different PR, though